lib/mavenlink/request.rb

```diff
@@ -140,13 +149,13 @@ def build(attributes)
     # @param attributes [Hash]
     # @return [Mavenlink::Response]
     def create(attributes)
-      perform { client.post(collection_name, collection_name.singularize => attributes) }
+      perform { client.post(collection_name, collection_name.singularize => scope_with_stringified_arrays(attributes)) }
```

Comment:
This is a good point. It's weird because they still enter the strange rails way `custom_field_value[value][]=1500&custom_field_value[value][]=USD`.

Reply:
I looked into this a little more. I was hoping that we'd be able to keep it for `GET` request filters. However, after looking into it I found at least [one filter](http://developer.mavenlink.com/beta/#tag/Account-Memberships%2Fpaths%2F~1account_memberships%2Fget) which does take an actual array (`user_ids[]=1&user_ids[]=2`).

So unfortunately, because our API is inconsistent in this area, I'm reverting this change.